tap_jira/streams.py

```diff
@@ -4,11 +4,48 @@
 import dateparser
 
 from singer import metrics, utils, metadata, Transformer
+from singer.transform import SchemaMismatch
+from dateutil.parser._parser import ParserError
 from .http import Paginator,JiraNotFoundError
 from .context import Context
 
 DEFAULT_PAGE_SIZE = 50
 
+def handle_date_time_schema_miss_match(exception, record): # pylint: disable=inconsistent-return-statements
+    """
+    Handling exception for date-time value out of range.
+    """
+
+    if ("{'format': 'date-time', 'type': ['string', 'null']}"
+        in exception.args[0].split("\n\t")[1]):
+        nested_keys = (
+            exception.args[0].split("\n\t")[1].split(":")[0]
+        )
+        # Getting the date value from nested object
+        obj = record.copy()
```

Comment:
comment mismatch: We are not getting the date value here.

Reply:
Added proper comment